### PR TITLE
Show Debug Info

### PR DESCRIPTION
**Pull request by h0ngcha0**

### CODE REVIEW

File: packages/web3/src/contract/contract.ts [hunk 1]
```diff
@@ -1055,9 +1064,9 @@ export class Contract extends Artifact {
     return signerParams
   }
 
-  buildByteCodeToDeploy(initialFields: Fields): string {
+  buildByteCodeToDeploy(initialFields: Fields, isDevnet: boolean): string {
     try {
-      return ralph.buildContractByteCode(this.bytecode, initialFields, this.fieldsSig)
+      return ralph.buildContractByteCode(isDevnet ? this.bytecodeDebug : this.bytecode, initialFields, this.fieldsSig)
```
Comment (h0ngcha0) [inline packages/web3/src/contract/contract.ts:1069]: Deploy `bytecodeDebug` to devnet, so we can print the debug message.

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ services:
         condition: service_healthy
 
   alephium:
-    image: alephium/dev-alephium:2.8.0
+    image: liuhongchao/dev-alephium:test-debug
```
Comment (h0ngcha0) [inline docker/docker-compose.yml:34]: Will update when we have the new full node image.

File: packages/cli/src/deployment.ts [hunk 3]
```diff
@@ -317,8 +317,9 @@ function createDeployer<Settings = unknown>(
     params: DeployContractParams<P>,
     taskTag?: string
   ): Promise<DeployContractResult<T>> => {
+    const isDevnet = network.networkId !== 0 && network.networkId !== 1
```
Comment (polarker) [inline packages/cli/src/deployment.ts:320]: Maybe we use `networkId % 3 == 2` following the Scala code. `%3 = 0, 1, 2` would be `mainnet, testnset, and devnet`.

Comment (polarker) [reply packages/cli/src/deployment.ts:320]: As discussed, let's keep this for now until the next network upgrade

File: docker/devnet.conf [hunk 4]
```diff
@@ -26,7 +26,7 @@ alephium.consensus.block-target-time = 10 millis
 alephium.consensus.uncle-dependency-gap-time = 0 seconds
 alephium.network.leman-hard-fork-timestamp = 1643500800000 // GMT: 30 January 2022 00:00:00
 
-alephium.network.network-id = 5
+alephium.network.network-id = 4
```
Comment (h0ngcha0) [inline docker/devnet.conf:29]: Will address the config issue in a seperate PR
This requires an update to full node devnet config template, which affected a lot of tests.

**Review by polarker — APPROVED** [review]
LGTM

**Pull request merged**